### PR TITLE
Update validator docs

### PR DESCRIPTION
**Pull request by kylezs**
- Move file permission changes before the final checks so the final check output matches the final expected outcome
- We now have validator gas refunds, so docs updated to reflect that

### CODE REVIEW

File: pages/validators/mainnet/validator-setup/keys.mdx [hunk 1]
```diff
@@ -61,7 +61,7 @@ Take special note of the **Validator Account ID** beginning with `cF`. This is t
 
 ### Setting Up Your Validator's Ethereum Account
 
-You must ensure that the public Ethereum address generated above has at least **0.2 ETH** on Ethereum Mainnet and **0.1 ETH** on Ethereum Arbitrum. Any fees your validator pays will be refunded back to the same account at the end of an epoch. You should monitor this balance to ensure it doesn't fall below the required amount.
+You must ensure that the public Ethereum address generated above has at least **0.2 ETH** on Ethereum Mainnet and **0.1 ETH** on Ethereum Arbitrum. At the end of every epoch, an attempt to refund the validator fees occurs. Only if the total refunded amount is 100x the fee it would cost to do the refund does the refund actually occur. You should monitor this balance to ensure it doesn't fall below the required amount.
```
Comment (dandanlen) [inline pages/validators/mainnet/validator-setup/keys.mdx:64]: FWIW I think this level of detail is bit unnecessary and only adds confusion - we could just say that the protocol regularly pays out gas refunds to this address, but that it should be monitored anyway. The exact conditions or rules of the refund are not very relevant and subject to change.

Comment (kylezs) [reply pages/validators/mainnet/validator-setup/keys.mdx:64]: Yeah was thinking a bit the same, but the idea with providing the extra detail is so they don't think they can measure one epoch's worth of cost and then just use that, they need to have more than that to account for this 100x fee.

Comment (dandanlen) [reply pages/validators/mainnet/validator-setup/keys.mdx:64]: I don't really understand - if we provide the extra detail, doesn't it make it more likely that someone will decide to try to cut it a bit closer? 

Can't we just say that the protocol regularly tops up these accounts, and we recommend an initial deposit of X?

Comment (kylezs) [reply pages/validators/mainnet/validator-setup/keys.mdx:64]: Yeah I guess by keeping it vague as "regularly" it would achieve the same effect.

**Pull request merged**